### ISSUE TITLE
Play with Docker - A simple, interactive and fun playground to learn Docker

### ISSUE THREAD

**Issue by UjwalKandi** [UjwalKandi/Issue-Bookmarks#509]
**TL;DR** - PWD is a Docker playground which allows users to run Docker commands in a matter of seconds. It gives the experience of having a free Alpine Linux Virtual Machine in browser, where you can build and run Docker containers and even create clusters in [Docker Swarm Mode](https://docs.docker.com/engine/swarm/). Under the hood Docker-in-Docker (DinD) is used to give the effect of multiple VMs/PCs. In addition to the playground, PWD also includes a training site composed of a large set of Docker labs and quizzes from beginner to advanced level available at [training.play-with-docker.com](http://training.play-with-docker.com/).

**Website:**
https://labs.play-with-docker.com
